runtimes/deno-1.21/server.ts

```diff
@@ -21,14 +21,33 @@ app.use(async (ctx) => {
     payload: body.payload ?? ''
   };
 
+  const stdlog = console.log.bind(console);
+  const stderror = console.error.bind(console);
+  const stdinfo = console.info.bind(console);
+  const stddebug = console.debug.bind(console);
+  const stdwarn = console.warn.bind(console);
+  let logs: any[] = [];
+
+  console.log = console.error = console.info = console.warn = console.debug = function(){
+    var args:any[] = [];
+    Array.from(arguments).forEach(arg => {
+        if(arg instanceof Object || Array.isArray(arg)) {    
+            args.push(JSON.stringify(arg));
+        } else {
+            args.push(arg)
+        }
+    });
+    logs.push(args.join(" "));
+  }
+
   const response = {
     send: (text: string, status = 200) => {
       ctx.response.status = status;
-      ctx.response.body = text;
+      ctx.response.body = {response: text, stdout: logs.join('\n')};
```

Comment:
In the other runtimes ( c++, dart ) , we are sending the response body as a json encoded string. Here we're directly sending a JSON. I don't think this is expected.

Reply:
Please test all the runtimes to **verify** this is working

Reply:
I think we should add new runtimes in deno and discard the old ones. I'll look at the releases, the old ones I don't think it's worth spending much time on them, so I've commented those out.

Reply:
In that case, let's remove the older ones and finalise this PR